### ISSUE TITLE
ControlNet no longer working

### ISSUE THREAD

**Issue by swisskris** [volotat/SD-CN-Animation#188]
Latest ControlNet update doesn't work with SD-CN-Animation Extension v0.9.

<img width="811" alt="Screenshot 2023-08-07 at 17 02 50" src="https://github.com/volotat/SD-CN-Animation/assets/123172512/0a1300f7-40dd-4d31-8882-9ca4b618140f">

**Comment by swisskris** [comment]
Only solution is to disable the SD-CN extension and then I can use ControlNet iwith other extensions such as Img2Img

**Comment by vapidvim** [comment]
@swisskris, it actually conflicts with a lot of extensions. I recommend disabling it when not using it and vice versa for all the extensions that show errors in the console when running it. 
Volotat must be busy with life, but I also look forward to continued updates as this is one of the best animation extensions for a1111.

**Comment by swisskris** [comment]
Yeah, I have it disabled. @volotat hope you still maintain this extension as it is absolutely the best video extension right now

**Comment by Nishkriya** [comment]
As far as I know, SD-CN Animation from mariaWitch at least not interferes with ControlNet by mattyamonaka. Didnt yet try correctness of SD-CN  though

**Comment by deeplearn-art** [comment]
> Hope development hasn't stopped on this great extension as last version update seems to be from May

I believe the extension has been abandoned https://github.com/volotat/SD-CN-Animation/issues/180#issuecomment-1663056643

**Comment by Duemellon** [comment]
> 

Are you saying there's another SD-CN out in the world? Where can I find this? I tried searching for SD-CN Animation from mariaWitch but Google can't find it.

**Comment by Nishkriya** [comment]
Why, I found it in forks right here.

**Comment by dkamajix** [comment]
SD CN extension has conflicts with latest controlnet unfortunately!
I get this when using img2img with controlnet with the extension off.
If I remove SDCN controlnet then works fine.
Has anyone Found a fix, I love SD CN - so much easier than Deforum!

2023-09-02 21:45:17,260 - ControlNet - WARNING - No ControlNetUnit detected in args. It is very likely that you are having an extension conflict.Here are args received by ControlNet: ().

**Comment by joshua76y** [comment]
> SD CN extension has conflicts with latest controlnet unfortunately! I get this when using img2img with controlnet with the extension off. If I remove SDCN controlnet then works fine. Has anyone Found a fix, I love SD CN - so much easier than Deforum!
> 
> 2023-09-02 21:45:17,260 - ControlNet - WARNING - No ControlNetUnit detected in args. It is very likely that you are having an extension conflict.Here are args received by ControlNet: ().

Same problem as you, also like this extension very much, hope the author can fix this problem

**Comment by Duemellon** [comment]
> Same problem as you, also like this extension very much, hope the author can fix this problem

At this point AnimateDiff has pretty much replaced this & provides more features.  I do hope this programmer has moved on to more things involving the new animation stuff. This was groundbreaking at the time.

**Comment by ricardofeynman** [comment]
@Duemellon Respectfully, hard disagree. SD-CN still has its own unique aesthetic. Great we can all have prompt travelling, seamless loops, bespoke ComfyUI workflows with AnimateDiff and the results can be incredible, but there's no substitute (yet) for the look and feel of SD-CN generations with the right combination of prompt, parameters, and model. FloweR seems a unique approach that should be revisited. This is gold.
